### PR TITLE
{lib}[GCCcore/10.3.0] UCX-CUDA v1.10.0

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

Draft'ish. Depends on new easyblock

- [x] ~https://github.com/easybuilders/easybuild-easyblocks/pull/2491~

### CODE REVIEW

File: easybuild/easyconfigs/u/UCX-CUDA/UCX-CUDA-1.10.0-GCCcore-10.3.0-CUDA-11.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('UCX', version),
+    ('CUDAcore',  '11.3.1', '', True),
+    ('GDRCopy', '2.2'),
+]
+
+sanity_check_commands = ["ucx_info -d"]
```
Comment (boegel) [inline easybuild/easyconfigs/u/UCX-CUDA/UCX-CUDA-1.10.0-GCCcore-10.3.0-CUDA-11.3.1.eb:42]: This can be removed, it's handled by the easyblock

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr=2491"

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2491
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/090681b7910b262729e0c1c78999f4a6 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2491
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/dcf41d0a091c730fd0a1167f777b6528 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr=2491"

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2491
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3517.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/01342f113f2c5e0014fbc17cf611d2a9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2491
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2603.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/eba3755bac7d07fdf0b057e65550d22c for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2491
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos8-eb - Linux centos linux 8.3.2011, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/7384e931f7936c48122b0ea249388b9f for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2491
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/3b1b1a6199c816881c8718458aeb0079 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3615.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/82d82478c8e80a0b3971b2bc36cd3aad for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**